_inc/client/state/settings/reducer.js

```diff
@@ -201,3 +201,12 @@ export function appsCardDismissed( state ) {
 export function emptyStatsCardDismissed( state ) {
 	return get( state.jetpack.settings.items, 'dismiss_empty_stats_card', false );
 }
+
+/**
+ * Returns true if Empty Stats card has been dismissed.
```

Comment:
This description for `showWelcomeForNewPlan ` is weird

Reply:
bad copy pasta.  fixed in 6c5c299ae